samtranslator/translator/translator.py

```diff
@@ -332,6 +332,15 @@ def _get_embedded_connectors(self, resources: Dict[str, Any]) -> List[Resource]:
                         connector_dict,
                     )
 
+                    # In case the generated resource already contains Metadata, we need to append to it
+                    try:
+                        original_metadata = generated_connector.get_resource_attribute("Metadata")
+                    except KeyError:
+                        original_metadata = {}
+                    generated_connector.set_resource_attribute(
+                        "Metadata", {**original_metadata, "aws:sam:connectors:type": "Embedded"}
+                    )
+
```

Comment:
Can we use [suppress](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) here? Something like:
```
original_metadata = {}
suppress(KeyError):
    original_metadata = generated_connector.get_resource_attribute("Metadata")
generated_connector.set_resource_attribute(
    "Metadata", {**original_metadata, "aws:sam:connectors:type": "Embedded"}
)
```